From 1f91efbfe985a8f4c441acf38f4061b2bed771de Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa@fedoraproject.org>
Date: Mon, 22 Jan 2024 18:07:14 -0500
Subject: [PATCH] Revert "kwinrc: Disable the Blur plugin in kwin by default"

This is now fixed in Plasma 6.

This reverts commit dcee55e77a73a24fe254ba9e7e602e748667204a.
---
 usr/share/kde-settings/kde-profile/default/xdg/kwinrc | 2 --
 1 file changed, 2 deletions(-)
 delete mode 100644 usr/share/kde-settings/kde-profile/default/xdg/kwinrc

diff --git a/usr/share/kde-settings/kde-profile/default/xdg/kwinrc b/usr/share/kde-settings/kde-profile/default/xdg/kwinrc
deleted file mode 100644
index 1a9bebd..0000000
--- a/usr/share/kde-settings/kde-profile/default/xdg/kwinrc
+++ /dev/null
@@ -1,2 +0,0 @@
-[Plugins]
-blurEnabled=false
-- 
2.43.0

